### ISSUE TITLE
Staging Accessibility Review Findings for Find VA Forms Search MVP

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8247]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

## Point of Contact

**VFS Point of Contact:** Jennifer

**Comment by jenstrickland** [comment]
@ncksllvn Wrapping up the staging accessibility review, and I want to make sure I properly assign the issue tickets. 

- #8252 and #8253, the h4 with the text all as a single link: is this a CMS or design system component that VSP needs to fix, or is this something for your team to remediate?

- #8254 and #8255, the pagination focus outline issues: are these specific to this implementation, or is this a sitewide, design system issue for VSP to address?

**Comment by jenstrickland** [comment]
@1Copenut This staging accessibility review is ready for your VSP review.

**Comment by ncksllvn** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/8252 - Already on our radar actually in #8135 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/8253 - This is from a CMS template so it's site-wide. 

The pagination issues are part of the React widget but we're just using a design system component - https://design.va.gov/components/pagination - no styling done on our end for that

**Comment by 1Copenut** [comment]
@jenstrickland This is a very comprehensive review! I have a couple of items I wanted to run past you and see if they rose to the level of issues:

1. The search results text is listed as an `<h2>` but it doesn't feel like a heading. I'd be okay with it staying a heading, or being flipped to a `<p>`
1. The search results text doesn't need the yellow focus halo. I've been leaning toward only adding the halo on interactive elements, not focus targets.
1. The blue cards *could* have `<h3>` instead of just plain text? Screenshot below.

---
![Screen Shot 2020-04-22 at 10 21 36 AM](https://user-images.githubusercontent.com/934879/80000784-33a11b80-8483-11ea-9a6b-f9afb3340851.png)

**Comment by ncksllvn** [comment]
> The search results text doesn't need the yellow focus halo. I've been leaning toward only adding the halo on interactive elements, not focus targets.

We should really figure out a pattern for this. If we focus on an element on screen that isn't a default focusable element (button, link, etc) then we get that gold outline. We've been overriding this on a case-by-case basis. Search `outline:` throughout vets-website to see what I mean. Personally and as a dev, I like the gold outline though 🙂 

> The blue cards could have `<h3>` instead of just plain text? Screenshot below.

This markup was just copied from https://staging.va.gov/pittsburgh-health-care/health-services/. It's going to be a Drupal template soon but isn't right now because it only works on VAMC pages. I would prefer that we address these blue boxes in this ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/49130 - because anything we do with the blue boxes specific to Find Forms will just be wiped once Drupal is housing this data.

**Comment by jenstrickland** [comment]
@1Copenut 

> The search results text is listed as an `<h2>` but it doesn't feel like a heading. I'd be okay with it staying a heading, or being flipped to a `<p>`.

Visual design has the weight of it minimized, and that's okay but not ideal in my opinion, but serving to screen readers as a heading is helpful when traversing by headings. I'd prefer it was visually a heading, but choosing priorities. 😉 

> The search results text doesn't need the yellow focus halo. I've been leaning toward only adding the halo on interactive elements, not focus targets.

I wasn't sure why the yellow focus halo was applied here. I value it on focus targets, honestly, for cognitive clarity. Should I document as an issue, or is it still an open item for global guidelines?

> The blue cards could have `<h3>` instead of just plain text? Screenshot below.

I agree that the list items of the blue cards could have headings, but wasn't sure if the CMS component had that flexibility, and this read fine to me.

**Comment by jenstrickland** [comment]
@ncksllvn — please do not change the search results heading to a paragraph tag. 😄

**Comment by jenstrickland** [comment]
@ncksllvn 
> #8252 - Already on our radar actually in #8135

Does this count as a CMS template issue, though? If so, I want to make sure to log it, so that other teams don't inherit it.

**Comment by 1Copenut** [comment]
To confirm and align my feedback with @jenstrickland 

1. The search results string is staying a heading 👍 
1. The blue cards are fine as is 👍
1. The yellow halo should be removed from the search results string. This is a low-priority item and ultimately should be captured in our VA.gov design guide online. 

Thanks for great conversation and real-time feedback Jennifer and @ncksllvn !

**Comment by ncksllvn** [comment]
> Does this count as a CMS template issue, though? If so, I want to make sure to log it, so that other teams don't inherit it.

This is specific to the Find Forms React widget, so no, not CMS or site-wide.

**Comment by jenstrickland** [comment]
@stephorkandatx The staging accessibility review is complete, and logged in the [508 Product Review List](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-product-review-list.md). 

Definition of done:
Ensure all issues are resolve in the coming 3 sprints, beginning with the 508-defect-2 issues which should be remediated in the next sprint.

**Comment by stephorkandatx** [comment]
Thank you @jenstrickland I will review these today. If I have any questions, I will reach out.

**Comment by MarciMcGuireGCIO** [comment]
Moving this from Public Websites to Decision Tools team, Find a VA Form Product
